docs/Overview.md

```diff
@@ -0,0 +1,213 @@
+      connectOwnerToStore: function () {
+        return {
+          store: roomStore,
+          listener: function () {
+            this.setState({
+              rooms: roomStore.getAll()
+            });
+          }.bind(this)
+        };
+      },
+
+      //TuxOwnerClasses can automatically share static props and methods with any level of child component as long as there is not another OwnerClass between the two. In this case all children beneath the RoomViewOwner will automatically have access to the delete and edit methods under this.nearestOwnerProps
+      registerOwnerProps: function () {
+        return {
+          delete: roomActions.delete,
+          edit: roomActions.edit
+        };
+      },
+
+      componentDidMount: function () {
+        roomActions.get();
+      },
+
+      render: function () {
+        return (
+          <RoomCreateForm />
+          <Rooms rooms={this.state.rooms} />
+        );
+      }
+    });
+
+    module.exports = RoomViewOwner;
+```
+
+`Tux` even provides a class designed to take advantage of the performance boosting tools in `React` for performance critical components in your application.
+
+```
+    var React = require('tux/React');
+
+    var Room = React.createMutableClass({
+      propTypes: {
+        room: React.PropTypes.object.isRequired
+      },
+
+      //just as with standard React props we can check to make sure we have the nearestOwnerProps we will need for our component
+      nearestOwnerPropTypes: {
+        delete: React.PropTypes.func.isRequired
+      },
+
+      //mutableTraits allows us to define the props and state that will be able to update in this class. We perform a one time deep search to find the props/state and, from then on, whenever the component is triggered to update it checks to see if the props or state have changed and, if they have not, then the class does not need to rerender in the virtual DOM
+      mutableTraits: {
+        props: 'name'
+      },
+
+      handleDelete: function (e) {
+        e.preventDefault();
+
+        //using the delete method we automatically get access to from our nearest owner
+        this.nearestOwnerProps.delete(this.props.room);
+      },
+
+      render: function () {
+        return (
+          <div>
+            <h2>{this.props.room.name}</h2>
+            <button onClick={this.handleDelete}>Delete</button>
+          </div>
+        );
+      }
+    });
+```
+
+## Next Steps
+Hopefully now you are a little excited about `TuxedoJS` and how you can use it in your apps. From here you can head to [getting started](./getting-started) to build a basic Todo app in `tux`. Alternatively, take a look at our deployed [Tux Chat App](TuxedoJS.github.io/TuxChatApp) and [Tux Todo App](TuxedoJS.github.io/TuxTodoApp). Lastly, check out our [api documentation](./docs) to get a deep dive with `Tux`.
```

Comment:
Recommend changing lower case \`tux\` to uppercase \`Tux\` as it is used as the last word.